routes/stats.tsx

```diff
@@ -122,17 +75,52 @@ export default function StatsPage(props: PageProps<StatsPageData>) {
   );
 
   return (
-    <main class="flex-1 p-4">
-      <div class="gap-4">
-        {charts.map((chart) => (
+    <main class="flex-1 p-4 grid gap-12 md:grid-cols-2">
```

Comment:
thought it would be a neat idea to have the charts rendered as grid again.

Reply:
I'd rather each take up one line and be much bigger, but my preference isn't strong.

Reply:
please let me know if i should remove the grid.